homeassistant/auth/mfa_modules/totp.py

```diff
@@ -87,7 +63,7 @@ def __init__(self, hass: HomeAssistant, config: dict[str, Any]) -> None:
     @property
     def input_schema(self) -> vol.Schema:
         """Validate login flow input data."""
-        return vol.Schema({vol.Required(INPUT_FIELD_CODE): str})
+        return vol.Schema({vol.Required(INPUT_FIELD_CODE): int})
```

Comment:
Authy shows the code separated by a space after the third letter. Example `123 456`
What would happen if the user inserts the code `123 456`? Is this code valid, or will the validation fail?